docs/src/tutorials/tutorial_2.md

```diff
@@ -81,48 +81,62 @@ The following invocation will create the head:
 ...
 ```
 
-### Different types of defining an Action
+An `Action` consists of at least one part, namely calling a function which draw something on to the canvas. 
```

Comment:
"draw" -> "draws", right? (Always feel awkward to correct your English :smile: )

Reply:
Haha, good catch!